.prettierrc

```diff
@@ -0,0 +1 @@
+'trailingComma': 'none'
```

Comment:
```suggestion
'trailingComma': 'es5'
```

(this would reduce the diff, if we go the trailingComma option)

Reply:
Then we need to agree n a new convention that we work with trailing , - @mouse-reeve, what do you say?